RTRDoubleArmV7:
  rtr_double_arm:
    joints:
      - "MFRAME"
      - "BLOCK"
      - "BOOM"
      - "ARM"
      - "TOHKU_PITCH"
      - "TOHKU_ROLL"
      - "UFRAME"
      - "MNP_SWING"
      - "MANIBOOM"
      - "MANIARM"
      - "MANIELBOW"
      - "YAWJOINT"
      - "HANDBASE"
      - "TOHKU_TIP_01"
      - "TOHKU_TIP_02"
      - "PUSHROD"

  tohoku_gipper:
    group_name:
      - tohoku_arm
    link_name:
      - TOHKU_ROLL
    joints:
      - TOHKU_TIP_01
      - TOHKU_TIP_02

  mani_gipper:
    group_name:
      - mani_arm
    link_name:
      - HANDBASE
    joints:
      - PUSHROD